src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq_vmr.c

```diff
@@ -431,13 +431,14 @@ static size_t xgq_vmr_log_dump(struct xocl_xgq_vmr *xgq, int num_recs, char *buf
 				sizeof(log));
 			log_idx = (log_idx + 1) % VMR_LOG_MAX_RECS;
 
-			/* calling call back function */
-			count += dump_cb(xgq, buf ? buf + count : NULL, log.log_buf);
```

Comment:
since we are copying from log.log_buf, can you use the sizeof(log_buf)? I know it is the same, but it is not intuitive for any other developers to understand what is the size.

Reply:
and it is natural to use count + sizeof(log.log_buf) < PAGE_SIZE